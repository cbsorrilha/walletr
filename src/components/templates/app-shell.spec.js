import React from 'react'
import { render } from '@testing-library/react'
import AppShell from './app-shell';

describe("Test the app-shell component", () => {
  it('should match the snapshots', () => {
    const component = render(<AppShell />);
    expect(component).toMatchSnapshot();
  })
});