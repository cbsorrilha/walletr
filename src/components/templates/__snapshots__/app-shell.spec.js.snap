// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the app-shell component should match the snapshots 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="sc-bxivhb iESyTA sc-bdVaJa cTyYxe"
        >
          <div
            class="sc-bdVaJa bZTbev"
            direction="column"
            width="90%"
          >
            <h1
              class="sc-bwzfXH fwGeXu"
            >
              Wallet
              <span
                class="sc-htpNat kUfHpA"
              >
                r
              </span>
            </h1>
          </div>
        </div>
        <div
          class="sc-bdVaJa cMTSTG"
          direction="column"
          width="90%"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="sc-bxivhb iESyTA sc-bdVaJa cTyYxe"
      >
        <div
          class="sc-bdVaJa bZTbev"
          direction="column"
          width="90%"
        >
          <h1
            class="sc-bwzfXH fwGeXu"
          >
            Wallet
            <span
              class="sc-htpNat kUfHpA"
            >
              r
            </span>
          </h1>
        </div>
      </div>
      <div
        class="sc-bdVaJa cMTSTG"
        direction="column"
        width="90%"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
